cpr/session.cpp

```diff
@@ -486,6 +486,17 @@ void Session::Impl::SetSslOptions(const SslOptions& options) {
         if (!options.key_pass.empty()) {
             curl_easy_setopt(curl_->handle, CURLOPT_KEYPASSWD, options.key_pass.c_str());
         }
+    } else if (!options.key_blob.empty()) {
+      struct curl_blob blob;
+      blob.data = const_cast<char *>(options.key_blob.c_str());
```

Comment:
Don't use a `const_cast` here. A solution would be to store a mutable copy of `key_blob` in `KeyBlob`.

Reply:
Requested changes resolved in d68d6ee.